plugins/hydra_optuna_sweeper/setup.py

```diff
@@ -26,10 +26,8 @@
         "Development Status :: 4 - Beta",
     ],
     install_requires=[
-        # TODO(toshihikoyanase): Change version restriction to hydra-core>=1.1 after hydra-core=1.1.0 is released.
         "hydra-core>=1.1.0.dev7",
-        "optuna<2.5.0",
-        "numpy<1.20.0",  # remove once optuna is upgraded to support numpy 1.20
+        "optuna",
```

Comment:
indicate the compatible version (where _tell was deprecated? maybe the latest?)
```suggestion
        "optuna>=X",
```

Reply:
Optuna v2.5.0 deprecated `study._ask` and `study._tell` and introduced `study.ask` and `study.tell` (c.f., https://github.com/optuna/optuna/pull/2158).

```suggestion
        "optuna>=2.5.0",
```